CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.5)
 
 project(libtsm
     LANGUAGES C
-    VERSION 4.0.2
+    VERSION 4.1.0
```

Comment:
Please drop this as I try to be consistent and only bump the version right before release.

Reply:
Hmm good question, I haven't considered that. Just leave it as is for simplicity.